test/infrastructure/docker/cloudinit/writefiles.go

```diff
@@ -29,6 +29,21 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const (
+	kubeadmInitPath          = "/run/kubeadm/kubeadm.yaml"
+	kubeproxyComponentConfig = `
+---
+apiVersion: kubeproxy.config.k8s.io/v1alpha1
+kind: KubeProxyConfiguration
+metadata:
+  name: config
```

Comment:
metadata is not required for component config files

Reply:
was wondering about that, it was in the kind version so i copied it over.

Reply:
Updated